.github/workflows/benchmark.yml

```diff
@@ -160,14 +160,14 @@ jobs:
   # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 
   performance_benchmark:
-    # put the benchmark in single job to keep constant environment during test
+    # put the benchmark in single job to keep constant environment during test python 3.8 is not applicable
     needs: [ download_data ]
 
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        python-version: [ "3.10", "3.9", "3.10", "3.11" ]
+        python-version: [ "3.9", "3.10", "3.11" ]
```

Comment:
If the benchmark is not available on python 3.8, then should we consider to set minimum version of CredSweeper as 3.9?

Reply:
There only speed benchmark limitation. CredSweeper can work wit 3.8.
As i see, https://pypistats.org/packages/credsweeper - the version is most popular. 
With any limitation of a package we will set the minimum to 3.9

Reply:
Okay. 3.8 is the most popular version...